docs/src/modules/components/TopLayoutCompany.js

```diff
@@ -31,17 +36,29 @@ function TopLayoutCompany(props) {
   const { description, rendered, title } = docs.en;
 
   return (
-    <AppFrame disableDrawer>
+    <BrandingProvider>
+      <AppHeader />
       <Head title={`${title} - MUI`} description={description} />
       <div className={classes.root}>
         <AppContainer component="main" className={classes.container}>
+          <Link
```

Comment:
We need to use the Next.js's Link component to retain the client-side navigation behavior

Reply:
Done!